modules/api-gateway/variables.tf

```diff
@@ -54,7 +54,7 @@ variable "orca_catalog_reporting_invoke_arn" {
 variable "api_gateway_stage_name" {
   type        = string
   description = "stage name for the ORCA cumulus reconciliation api gateway"
-  default     = "orca"
+  default     = "api"
```

Comment:
I now get this error while deploying 

```
│ Error: deleting API Gateway Deployment (dbtt7b): BadRequestException: Active stages pointing to this deployment must be moved or deleted

```

Reply:
I still get the same error. Does this change update the API pathing I posted above in the screenshot?

Reply:
Yes it will change it from `https://<api-id>.execute-api.us-west-2.amazonaws.com/orca/orca/datamanagement/reconciliation/internal/jobs` to `https://<api-id>.execute-api.us-west-2.amazonaws.com/api/orca/datamanagement/reconciliation/internal/jobs`

Reply:
Not sure why I am getting the stage error. Might be related to https://bugs.earthdata.nasa.gov/browse/ORCA-769

If you tested and verified that this works, then I think we are good.